### PR TITLE
Domain-request content updates (Part 2)

### PR DESCRIPTION
**Pull request by Katherine-Osos**
## Ticket

Resolves https://github.com/cisagov/manage.get.gov/issues/1240 (partially)

## Changes

Updated content within request flow to match updates made in the corresponding section of the Google doc:

- [.gov domain](https://docs.google.com/document/d/1GTTeRvKfe3lA_w2U4X_CBk8Q7BpMe7W0m_eVZsTHzEE/edit#heading=h.3tbugp1)

- [Purpose of your domain](https://docs.google.com/document/d/1GTTeRvKfe3lA_w2U4X_CBk8Q7BpMe7W0m_eVZsTHzEE/edit#heading=h.2u6wntf)

- [Your contact info](https://docs.google.com/document/d/1GTTeRvKfe3lA_w2U4X_CBk8Q7BpMe7W0m_eVZsTHzEE/edit#heading=h.2dlolyb)

- [Anything else](https://docs.google.com/document/d/1GTTeRvKfe3lA_w2U4X_CBk8Q7BpMe7W0m_eVZsTHzEE/edit#heading=h.3q5sasy)
- [Requirements for operating .gov](https://docs.google.com/document/d/11hSLhGul3_54UItgN0IG9K6FUA8mGTIzSoTMbVcdnG4/edit) 
-- Note: the body copy has been updated via this PR; however, the revised page title is being updated separately: #1639 

- - - - - - -
Review changes on my sandbox: [ko](https://getgov-ko.app.cloud.gov/)

### CODE REVIEW

**Comment by Katherine-Osos** [comment]
@h-m-f-t You've already reviewed this content in the Google docs, but I wanted to flag this PR in case you wanted to see the updated text for "**Requirements for operating a .gov**" in context.

File: src/registrar/templates/application_dotgov_domain.html [hunk 1]
```diff
@@ -41,10 +39,7 @@ <h2 class="margin-top-3">Domain examples for your type of organization</h2>
       <h2>What .gov domain do you want?</h2>
```
Comment (gabydisarli) [inline src/registrar/templates/application_dotgov_domain.html:39]: @Katherine-Osos I see you haven't changed this, but I'm noticing something funny in the way this h2 is displaying vs the h2 "Domain examples for your type of organization" 

they are two different sizes. The "What .gov domain you want?" h2 is 25.44px and the h2 "Domain examples for your type of organization" is at 24px. It seems like one may be getting optical sizing and the other one is a static 24px.  Might be a question for someone who knows the CSS though. @rachidatecs ? or @erinysong ?

Comment (gabydisarli) [reply src/registrar/templates/application_dotgov_domain.html:39]: **What .gov domain do you want?**
Screenshot 2024-01-17 at 10 07 44 AM


**Domain examples for your type of organization**
Screenshot 2024-01-17 at 10 12 54 AM

File: src/registrar/templates/application_dotgov_domain.html [hunk 2]
```diff
@@ -2,24 +2,22 @@
...
     <ul class="usa-list">
       <li>Be available </li>
-      <li>Be unique </li>
       <li>Relate to your organization’s name, location, and/or services </li>
       <li>Be clear to the general public. Your domain name must not be easily confused
       with other organizations.</li>
     </ul>
   </p>
 
+  <p>Names that uniquely apply to your organization are likely to be approved over names that could also apply to other organizations. In most instances, this requires including your state’s two-letter abbreviation.</p>
```
Comment (gabydisarli) [inline src/registrar/templates/application_dotgov_domain.html:14]: These paragraphs are missing the italicized sections

File: src/registrar/templates/includes/domain_example.html [hunk 3]
```diff
@@ -55,45 +54,49 @@
...
-    <li>Erie.gov</li>
+    <li>LACounty.gov</li>
     <li>LivingstonParishLA.gov</li>
     <li>MitchellCountyNC.gov</li>
+    <li>MiamiDade.gov</li>
   </ul>
 
 {% elif organization_type == 'city' %}
   <p>Most city domains must include the two-letter state abbreviation or clearly spell out the state name. Using phrases like “City of” or “Town of” is optional.</p>
   <p>Cities that meet one of the criteria below don’t have to refer to their state in the domain name.
```
Comment (gabydisarli) [inline src/registrar/templates/includes/domain_example.html:70]: The google doc has a slight change here:

> Cities that meet one of the criteria below don’t have to refer to their state in **their** domain name.

File: src/registrar/templates/application_requirements.html [hunk 4]
```diff
@@ -3,51 +3,55 @@
...
+<p>The .gov domain space exists to support a broad diversity of government missions. Generally, we don’t review or audit how government organizations use their registered domains. However, misuse of a .gov domain can reflect upon the integrity of the entire .gov space. There are categories of misuse that are statutorily prohibited or abusive in nature.</p>
 
 
-  <h2>What you can’t do with .gov domains</h2>
+  <h2>What you can’t do with a .gov domain</h2>
 
   <h3>Commercial purposes</h3>
 
-  <p>.Gov domains must not be used for commercial purposes, such as advertising that benefits private individuals or entities.</p>
+  <p>.A .gov domain must not be used for commercial purposes, such as advertising that benefits private individuals or entities.</p>
```
Comment (gabydisarli) [inline src/registrar/templates/application_requirements.html:13]: There's an extra period at the beginning of this sentence

File: src/registrar/templates/application_anything_else.html [hunk 5]
```diff
@@ -2,7 +2,9 @@
 {% load field_helpers %}
 
 {% block form_instructions %}
-  <p>Is there anything else you'd like us to know about your domain request? This question is optional.</p>
+    <h2>Is there anything else you'd like us to know about your domain request?</h2>
```
Comment (gabydisarli) [inline src/registrar/templates/application_anything_else.html:5]: change prime mark to apostrophe in "you'd"

**Comment by gabydisarli** [comment]
@Katherine-Osos I've finished reviewing, let me know when it's ready to go again!

File: src/registrar/templates/application_anything_else.html (hunk 5 above)
Comment (Katherine-Osos) [reply src/registrar/templates/application_anything_else.html:5]:
```suggestion
    <h2>Is there anything else you’d like us to know about your domain request?</h2> 
```

File: src/registrar/templates/application_dotgov_domain.html (hunk 2 above)
Comment (Katherine-Osos) [reply src/registrar/templates/application_dotgov_domain.html:14]:
```suggestion
  <p>Names that _uniquely apply to your organization_ are likely to be approved over names that could also apply to other organizations. In most instances, this requires including your state’s two-letter abbreviation.</p> 
```

File: src/registrar/templates/includes/domain_example.html (hunk 3 above)
Comment (Katherine-Osos) [reply src/registrar/templates/includes/domain_example.html:70]:
```suggestion
  <p>Cities that meet one of the criteria below don’t have to refer to their state in their domain name.
```

File: src/registrar/templates/application_requirements.html [hunk 6]
```diff
@@ -50,7 +50,7 @@ <h2>Domain renewal</h2>
   
   <p>.Gov domains are registered for a one-year period. To renew your domain, you'll be asked to verify your organization’s eligibility and your contact information. </p>
   
-  <p>Though a domain may expire, it will not be automatically put on hold or deleted. We’ll make extensive efforts to contact your organization before holding or deleting a domain.</p>
+  <p>Though a domain may expire, it will not automatically be put on hold or deleted. We’ll make extensive efforts to contact your organization before holding or deleting a domain.</p>
```
Comment (gabydisarli) [inline src/registrar/templates/application_requirements.html:53]: I love this edit, way better!

**Review by gabydisarli — APPROVED** [review]
Looks great!

**Pull request merged**